### ISSUE TITLE
Color Library Update to V1.1

### ISSUE THREAD

**Issue by tsdobbs** [cclrobotics/ARTBot#96]
We would like to increase the number of colors that are available for bio-printing. This version should add at least one more color to the existing 5 E.coli-based colors (in addition to the 2 wild-types mentioned in [Issue 95](https://github.com/cclrobotics/ARTBot/issues/95)). More than one extra color is absolutely welcomed!

The deliverable is 1 or more new color-producing microbial strains brought to Counter Culture Labs and printed using BioArtBot. The strain must meet the following requirements:
- Safe for use in a BSL1 environment
- Grows on media that is easy to buy or make, with a strong preference for LB media
- If a plasmid is identified rather than a strain, it should be expressible in E.coli so that the transformation can be done at CCL
- Makes a cool color that we don't already have in our library

Some additional strain considerations:
- Our current set of colors grows for 48 hours, with the bulk of growth happening between approx hours 8 and 16. New strains may produce an interesting effect if they follow a different growth curve, but it may add process complications if, at 48 hours, they have either died or have not yet grown
- Our current colors are resistant to the antibiotic Chloramphenicol. New strains need not necessarily be resistant to the same antibiotic, but it would be more convenient
- We have been using E.coli because they're highly standardized, but there's no reason the new colors can't be produced in a different species. The choice of species should balance convenience and opportunity for an interesting artistic effect

Steps:

- [ ] Develop a list of candidate strains/plasmids based on requirements above
- [ ] Arrange for shipment of selected strains/plasmids to Counter Culture Labs
- [ ] If necessary, transform E.coli to produce colors from plasmids
- [ ] Test print with strains
- [ ] Make glycerol stocks from strains
- [ ] Add colors/strains to the color database on bioartbot.org

**Comment by tsdobbs** [comment]
Some possible leads for strains:

- Color generators from iGEM Cambridge https://2009.igem.org/Team:Cambridge/Project/Pigments
    - Based on this paper: https://aem.asm.org/content/74/4/1190
- Open Reporter Project (a set of 42 "reporter" genes, i.e. colors, that are available for free) https://stanford.freegenes.org/collections/open-genes/products/open-reporter-collection#description
    - Not all of these may be appropriate - and I believe some of them are actually duplicates of our existing colors - but likely at least a few of these would work well

**Comment by patrickjyoung** [comment]
Amino Labs for Potential Strains: https://amino.bio/products/engineer-it-kit?pr_prod_strat=copurchase&pr_rec_pid=9874335300&pr_ref_pid=4370477088908&pr_seq=uniform

**Comment by tsdobbs** [comment]
@patrickjyoung I just realized that the Amino Labs kits are $35 for **each** color. That's not unfair since they give you a whole kit, but we really just need the strain and that's a lot for a strain!
Still, it might be nice to compare their strains to the strains from other sources. If we had to pick one to start, which one should we get?

**Comment by tsdobbs** [comment]
Because it was easy and free, I ordered the Open Reporter Collection listed above to Counter Culture Labs. Let's test these out as soon as we can get into the lab.

**Comment by tsdobbs** [comment]
Here is a working document to list candidate strains: https://docs.google.com/document/d/11ZP0kij9Bj79kEi6HQBOZTnYW0b4K_aHcxq5vh7IOqw/edit?usp=sharing

**Comment by patrickjyoung** [comment]
> @patrickjyoung I just realized that the Amino Labs kits are $35 for **each** color. That's not unfair since they give you a whole kit, but we really just need the strain and that's a lot for a strain!
> Still, it might be nice to compare their strains to the strains from other sources. If we had to pick one to start, which one should we get?

@tsdobbs I was looking at the free reporter gene package and was thinking we should check those out first before we order any from amino labs. Thoughts?

**Comment by tsdobbs** [comment]
> > @patrickjyoung I just realized that the Amino Labs kits are $35 for **each** color. That's not unfair since they give you a whole kit, but we really just need the strain and that's a lot for a strain!
> > Still, it might be nice to compare their strains to the strains from other sources. If we had to pick one to start, which one should we get?
> 
> @tsdobbs I was looking at the free reporter gene package and was thinking we should check those out first before we order any from amino labs. Thoughts?

Yeah, that sounds sensible to me - let's put Amino Labs on the list as an option for the future

**Comment by tsdobbs** [comment]
We've got the 32 (not 42 as they advertised) strains from freegenes.org and were able to grow them on plates. They shipped them in non-expression strains though, so we need to subclone the genes into expression strains before doing anything else. Next step is to subclone and grow.
